### DISCUSSION TITLE
Stuck fetching grammars

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#5170] [General]
Hi,

when I tried to run `hx --grammer fetch`, it only kept showing `Fetching 123 grammars` without any more progress.

I have tried using different networking, but all turned out to be the same. It was installed via Homebrew on M1 Mac.

**Comment by archseer** [comment]
How long did you run it for? It's a bunch of fetching so it could take a couple of minutes

**Comment by ghost** [comment]
> How long did you run it for? It's a bunch of fetching so it could take a couple of minutes

Hello about 5 - 10 minutes without any progress
